instrumentation/kamon-play/src/main/scala/kamon/instrumentation/play/PlayServerInstrumentation.scala

```diff
@@ -68,6 +68,13 @@ class PlayServerInstrumentation extends InstrumentationBuilder {
   private def hasGenericFutureListener(): Boolean = {
     try { Class.forName("io.netty.util.concurrent.GenericFutureListener") != null} catch { case _ => false }
   }
+
+  /**
+    * When using play gRPC, we can reuse the instrumentation that is provided by the akka HTTP module, we just need to
+    * apply the operation name and make a sampling decision.
+    */
+  onType("play.grpc.internal.PlayRouter$$anon$1")
```

Comment:
Do you know whether this will work fine across all scala versions? :thinking:

Reply:
Currently play-grpc has been cross-compiled with scala 2.12 and 2.13.